### PR TITLE
Apply content model guidelines to Getting started tutorial

### LINKED ISSUE #8778: Apply docs.ms.com content guidelines to Get started with ASP.NET Core
Apply internal style guidelines for tutorials to this doc.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ca6139d4-022c-395a-0abf-539660550df1
* Version Independent ID: 3dd23f8a-300e-3f78-1dc5-8b759aabdccc
* Content: [Get started with ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/getting-started/?view=aspnetcore-2.1&tabs=windows#feedback)
* Content Source: [aspnetcore/getting-started/index.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/getting-started/index.md)
* Product: **aspnet-core**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes #8778

### CODE REVIEW

File: aspnetcore/getting-started/index.md [hunk 1]
```diff
@@ -1,27 +1,46 @@
 ---
 title: Get started with ASP.NET Core
 author: rick-anderson
-description: A quick tutorial that creates and runs a simple Hello World app using ASP.NET Core.
+description: A quick tutorial that creates and runs a simple Hello World app using ASP.NET Core. 
 ms.author: riande
 ms.custom: mvc
+ms.topic: tutorial
```
Comment (scottaddie) [inline aspnetcore/getting-started/index.md]: Remove the `ms.topic` metadata from the doc. This is applied globally via *docfx.json*: https://github.com/aspnet/Docs/blob/master/aspnetcore/docfx.json#L83. Currently, `ms.topic: conceptual` is being applied: https://github.com/aspnet/Docs/blob/master/aspnetcore/docfx.json#L83. Feel free to change this to `tutorial`, if that's the desired effect.

File: aspnetcore/getting-started/index.md [hunk 2]
```diff
@@ -56,99 +75,36 @@ This document provides steps for creating and running an ASP.NET Core app.
...
-   ```
-
-3. If you have installed a later SDK version on your machine, create a *global.json* file to select the 1.0.4 SDK.
-
-   ```json
-   {
-     "sdk": { "version": "1.0.4" }
-   }
-   ```
+* Browse to [http://localhost:5001](http://localhost:5001).  Click **Accept** to accept the privacy and cookie policy. This app doesn't keep personal information.
```
Comment (scottaddie) [inline aspnetcore/getting-started/index.md]: - There are 2 spaces after the 1st sentence.
- http --> https (this way, the user understands why they trusted the dev cert)

File: aspnetcore/getting-started/index.md [hunk 3]
```diff
@@ -56,99 +75,36 @@ This document provides steps for creating and running an ASP.NET Core app.
...
-5. Restore the packages.
+* Open *Pages/About.cshtml* and modify the page with the following highlighted markup:
 
-   ```console
-   dotnet restore
-   ```
+   [!code-cshtml[](sample/getting-started/about.cshtml?highlight=9)]
 
-6. Run the app.
+* Browse to [http://localhost:5001/About](http://localhost:5001/About) and verify the changes are displayed.
```
Comment (scottaddie) [inline aspnetcore/getting-started/index.md]: http --> https

File: aspnetcore/getting-started/index.md [hunk 4]
```diff
@@ -56,99 +74,36 @@ This document provides steps for creating and running an ASP.NET Core app.
...
+> * Enable local HTTPS.
+> * Run the project.
+> * Make a change.
 
-[!INCLUDE [next steps](~/includes/getting-started/next-steps.md)]
+To learn more about ASP.NET Core, see the introduction:
 
-::: moniker-end
+> [!div class="nextstepaction"]
+> [Introduction to ASP.NET Core](xref:index)
```
Comment (scottaddie) [inline aspnetcore/getting-started/index.md]: Nit: let's change this link to use the following syntax while you're in here:

```
<xref:index>
```

**Comment by tdykstra** [comment]
@Rick-Anderson Do you want to take a look?

**Pull request merged**